lib/testing/src/lib/core/pages/login.page.ts

```diff
@@ -183,7 +183,7 @@ export class LoginPage {
 
     goToLoginPage() {
```

Comment:
checkRegisterDisplayed() is a method that in case of failure gives a timeout

I would suggest refactoring such methods into something like isRegisterDisplayed() returning a boolean promise. This can then be used in the tests inside expects, and in case of failure this will avoid the timeout